src/envoy/utils/utils.cc

```diff
@@ -139,5 +139,21 @@ Status ParseJsonMessage(const std::string& json, Message* output) {
   return ::google::protobuf::util::JsonStringToMessage(json, output, options);
 }
 
+void CheckResponseInfoToStreamInfo(
+    const istio::mixerclient::CheckResponseInfo& check_response,
+    StreamInfo::StreamInfo& stream_info) {
+  static std::string metadata_key = "istio.mixer";
+
+  if (!check_response.response_status.ok()) {
+    stream_info.setResponseFlag(
+        StreamInfo::ResponseFlag::UnauthorizedExternalService);
+    ProtobufWkt::Struct metadata;
+    auto& fields = *metadata.mutable_fields();
+    fields["status"].set_string_value(
+        check_response.response_status.ToString());
```

Comment:
Why do we need the new attribute `istio.mixer` if we have `check.error_code` and `check.error_message` already? Can this be erased from reporting bag?

Reply:
See https://github.com/istio/istio/issues/10965 for context, it is for putting descriptive message in access log. We can erase this from reporting bag, is report collecting all dynamic metadata?

Reply:
I don't know, but it looks like all metadata is sent over. I don't think we want redundant data since it adds no value and has performance costs. Please delete this "istio.mixer" from reporting bag.

Reply:
https://github.com/istio/proxy/issues/2096